tests/test_phabricator_client.py

```diff
@@ -16,6 +16,8 @@
 from tests.canned_responses.phabricator.repos import *
 from tests.canned_responses.phabricator.errors import *
 
+pytestmark = pytest.mark.usefixtures('docker_env_vars')
```

Comment:
Importing the fixture won't apply it - you need to include a fixture in the module's `pytestmark` to apply it to every test in a file.